src/scripts/main.js

```diff
@@ -1 +1,80 @@
+/* eslint-disable no-shadow */
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.documentElement.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  // eslint-disable-next-line prefer-promise-reject-errors
+  window.setTimeout(() => reject('First promise was rejected'), 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  const a = new Promise((resolve) => {
+    document.documentElement.addEventListener('click', () => {
+      resolve();
+    });
+  });
+
+  const b = new Promise((resolve) => {
+    document.documentElement.addEventListener('contextmenu', () => {
+      resolve();
+    });
+  });
+
+  const result = Promise.race([a, b]);
+
+  result.finally(() => resolve('Second promise was resolved'));
```

Comment:
The use of `Promise.race` for the second promise is correct as it resolves when either 'click' or 'contextmenu' occurs. Ensure this is the intended behavior.